geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java

```diff
@@ -8429,6 +8432,8 @@ void clearRegionLocally(RegionEventImpl regionEvent, boolean cacheWrite,
     }
 
     RegionVersionVector myVector = getVersionVector();
+    System.out.println(Thread.currentThread().getName() + ": LocalRegion.clearRegionLocally region="
```

Comment:
you've got a debug println that needs to be removed

Reply:
Oh, it's from another on-going bug fix. I removed it.